Makefile

```diff
@@ -1,60 +1,141 @@
+# Current Operator version
+VERSION ?= 2.0.0-snapshot
+# Default bundle image tag
+BUNDLE_IMG ?= quay.io/kiegroup/kogito-cloud-operator:$(VERSION)
+# Options for 'bundle-build'
+ifneq ($(origin CHANNELS), undefined)
+BUNDLE_CHANNELS := --channels=$(CHANNELS)
 endif
+ifneq ($(origin DEFAULT_CHANNEL), undefined)
+BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
+endif
+BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
+
+# Image URL to use all building/pushing image targets
+IMG ?= quay.io/kiegroup/kogito-cloud-operator:$(VERSION)
+# Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
+CRD_OPTIONS ?= "crd:trivialVersions=true"
 
-ifeq ($(BUILD_VERBOSE),1)
-       Q =
+# Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
+ifeq (,$(shell go env GOBIN))
+GOBIN=$(shell go env GOPATH)/bin
 else
-       Q = @
+GOBIN=$(shell go env GOBIN)
 endif
 
-#export CGO_ENABLED:=0
+all: generate manifests docker-build
 
-.PHONY: all
-all: build
+# Run tests
+ENVTEST_ASSETS_DIR = $(shell pwd)/testbin
+test: fmt lint
+	./hack/go-test.sh
 
-.PHONY: mod
-mod:
-	./hack/go-mod.sh
+# Build manager binary
+manager: generate fmt vet
+	go build -o bin/manager main.go
 
-.PHONY: format
-format:
-	./hack/go-fmt.sh
+# Run against the configured Kubernetes cluster in ~/.kube/config
+run: generate fmt vet manifests
+	go run ./main.go
+
+# Install CRDs into a cluster
+install: manifests kustomize
+	$(KUSTOMIZE) build config/crd | kubectl apply -f -
 
-.PHONY: go-generate
-go-generate: mod
-	$(Q)go generate ./...
+# Uninstall CRDs from a cluster
+uninstall: manifests kustomize
+	$(KUSTOMIZE) build config/crd | kubectl delete -f -
 
-.PHONY: sdk-generate
-sdk-generate: mod
-	operator-sdk generate k8s
+# Deploy controller in the configured Kubernetes cluster in ~/.kube/config
+deploy: manifests kustomize
+	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
+	$(KUSTOMIZE) build config/default | kubectl apply -f -
 
-.PHONY: vet
-vet:
-	./hack/go-vet.sh
+# Generate manifests e.g. CRD, RBAC etc.
+manifests: controller-gen
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
 
-.PHONY: test
-test:
-	./hack/go-test.sh $(coverage)
+# Run go fmt against code
+fmt:
+	go mod tidy
+	./hack/addheaders.sh
+	./hack/go-fmt.sh
 
-.PHONY: lint
 lint:
 	./hack/go-lint.sh
-	#./hack/yaml-lint.sh
 
-.PHONY: build
-image_registry=
-image_name=
-image_tag=
-image_builder=
-build:
-	./hack/go-build.sh --image_registry ${image_registry} --image_name ${image_name} --image_tag ${image_tag} --image_builder ${image_builder}
+# Generate code
+generate: controller-gen
+	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
+	./hack/openapi.sh
+
+# Build the docker image
+IMAGE ?= $(IMG)
+BUILDER ?= podman
+docker-build:
+	$(BUILDER) build . -t ${IMAGE}
+
+# Push the docker image
+docker-push:
+	$(BUILDER) push ${IMG}
```

Comment:
Why docker build use parameter `IMAGE` but docker push use `IMG`

Reply:
True, will change everywhere to use `IMAGE` only